backend/migrate.ts

```
@@ -0,0 +1,486 @@
+                text: oldFeedItem.title ? `# ${oldFeedItem.title}\n\n${oldFeedItem.text}` : oldFeedItem.text,
+                author: oldFeedItem.author,
+                createdAt: oldFeedItem.createdAt
+            };
+
+            try {
+                await db.lSet('feedItems', i, JSON.stringify(newFeedItem));
+                logger.info(`Migrated feed item ${oldFeedItem.id}`);
+            } catch (err) {
+                logger.error(`Error setting feed item at index ${i}:`, err);
+                logger.error('Feed item data:', newFeedItem);
+                continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)